app/templates/reset_password.html

```diff
@@ -0,0 +1,44 @@
+{% extends "base.html" %}
+{% block content %}
+<div class="container">
+    <h5>Please choose a new password:</h4>
+        {% if message %}
+        <div class="alert" style="color: red;">{{ message }}</div>
+        {% endif %}
+        <form method="post">
+            <div class="row">
+                <label for="username" class="col-sm-2 col-form-label">Username</label>
+                <div class="col-auto">
+                    <input type="text" class="form-control" name="username" id="username" placeholder="Required"
+                        autofocus><br>
+                </div>
+            </div>
+            <div class="row">
+                <label for="emil" class="col-sm-2 col-form-label">password</label>
+                <div class="col-auto">
+                    <input type="text" class="form-control" name="password" id="password" placeholder="Required"><br>
+                </div>
+                <div class="col-auto">
+                    <span id="PasswordHelpInline" class="form-text">
```

Comment:
password-help-inline (also, you can use bootstrap's styling guide to add danger elements)